pandas/tests/io/parser/test_parsers.py

```diff
@@ -99,3 +102,14 @@ def read_table(self, *args, **kwds):
         kwds = kwds.copy()
         kwds['engine'] = self.engine
         return read_table(*args, **kwds)
+
+
+class TestParameterValidation(object):
+    def test_unknown_engine(self):
+        with tempfile.NamedTemporaryFile() as fp:
+            df = tm.makeDataFrame()
+            df.to_csv(fp.name)
```

Comment:
No.  There shouldn't be any tests in this file.  I made a comment <a href="https://github.com/pandas-dev/pandas/pull/16511#discussion_r118742624">here</a> about it.